test/autorest/generated/lrogroup/lrosads.go

```diff
@@ -211,6 +185,15 @@ func (client *lrosaDsOperations) delete202NonRetry400HandleResponse(resp *azcore
 	return &result, nil
 }
 
+// delete202NonRetry400HandleError handles the Delete202NonRetry400 error response.
+func (client *lrosaDsOperations) delete202NonRetry400HandleError(resp *azcore.Response) error {
+	err := CloudError{}
+	if err := resp.UnmarshalAsJSON(&err); err != nil {
+		return err
+	}
+	return err
+}
```

Comment:
Does this unmarshal correctly now?

Reply:
Oh was it broken someplace?

Reply:
If I recall correctly. Some testserver tests have errors that are not in the CloudError format

Reply:
I see, sadly I'm not surprised.  This shouldn't fix any of those cases, we'd either need to update the swagger or the test server so the definitions match.

Reply:
Yes probably. I'll still take a look when I go over the missing tests.